### PR TITLE
Updates project with SDK v3.0.1

### PR DESCRIPTION
**Pull request by AmitaiB**
Also adds the SDK framework to save users that step.

### CODE REVIEW

File: JWPlayer-Developer-Demo/ObjCPlayerViewController.m [hunk 1]
```diff
@@ -55,14 +55,16 @@ - (void)createPlayer
                        [JWSource sourceWithFile:@"http://content.bitsontherun.com/videos/bkaovAYt-52qL9xLP.mp4" label:@"270p Streaming"],
                        [JWSource sourceWithFile:@"http://content.bitsontherun.com/videos/bkaovAYt-DZ7jSYgM.mp4" label:@"720p Streaming"]];
     
-    config.image = @"http://content.bitsontherun.com/thumbs/bkaovAYt-480.jpg";
-    config.title = @"JWPlayer Demo";
+    config.image    = @"http://content.bitsontherun.com/thumbs/bkaovAYt-480.jpg";
```
Comment (karimMourra) [inline JWPlayer-Developer-Demo/ObjCPlayerViewController.m:58]: remove blanks

File: JWPlayer-Developer-Demo/ObjCPlayerViewController.m [hunk 2]
```diff
@@ -72,124 +74,127 @@ - (void)createPlayer
...
     NSArray *waterfallTags = @[@"bad tag", @"another bad tag", @"http://playertest.longtailvideo.com/adtags/preroll_newer.xml"];
-    
+
     //MARK: JWAdBreak
-    config.adSchedule = @[[JWAdBreak adBreakWithTags:waterfallTags offset:@"pre"],
+    adConfig.schedule = @[[JWAdBreak adBreakWithTags:waterfallTags offset:@"pre"],
                           [JWAdBreak adBreakWithTag:@"http://playertest.longtailvideo.com/adtags/preroll_newer.xml" offset:@"0:00:05"],
                           [JWAdBreak adBreakWithTag:@"http://demo.jwplayer.com/player-demos/assets/overlay.xml" offset:@"7" nonLinear:YES],
-//                          [JWAdBreak adBreakWithTag:@"http://playertest.longtailvideo.com/adtags/preroll_newer.xml" offset:@"5"],
+                          //                          [JWAdBreak adBreakWithTag:@"http://playertest.longtailvideo.com/adtags/preroll_newer.xml" offset:@"5"],
```
Comment (karimMourra) [inline JWPlayer-Developer-Demo/ObjCPlayerViewController.m:102]: why is this commented out ? If we don't need it we should delete it

File: JWPlayer-Developer-Demo/SwiftPlayerViewController.swift [hunk 3]
```diff
@@ -77,68 +79,71 @@ class SwiftPlayerViewController: UIViewController, JWPlayerDelegate {
...
+
         //MARK: Ad Schedule
-        config.adSchedule = [JWAdBreak(tags:waterfallTags as! [String], offset:"1"),
+        adConfig.schedule = [JWAdBreak(tags:waterfallTags as! [String], offset:"1"),
                              JWAdBreak(tag: "http://playertest.longtailvideo.com/adtags/preroll_newer.xml", offset:"5"),
-                             JWAdBreak(tag: "http://demo.jwplayer.com/player-demos/assets/overlay.xml", offset: "7", nonLinear: true),
-                             //                           JWAdBreak(tag: "http://playertest.longtailvideo.com/adtags/preroll_newer.xml", offset:"0:00:05"),
-            JWAdBreak(tag: "http://playertest.longtailvideo.com/adtags/preroll_newer.xml", offset:"50%"),
-            JWAdBreak(tag: "http://playertest.longtailvideo.com/adtags/preroll_newer.xml", offset:"post")]
+//                             JWAdBreak(tag: "http://demo.jwplayer.com/player-demos/assets/overlay.xml", offset: "7", nonLinear: true),
```
Comment (karimMourra) [inline JWPlayer-Developer-Demo/SwiftPlayerViewController.swift:107]: if we won't be using this, might as well delete it

**Review by karimMourra — CHANGES_REQUESTED** [review]
@AmitaiB I don't like the spacing but if you think it makes the demo easier to read for some of the users, that's fine; just don't do it in production code. 
I'm not sure i like the idea of including the SDK in the demo; @okulagin @coderDove @kobertynska any thoughts ?

File: JWPlayer-Developer-Demo/SwiftPlayerViewController.swift (hunk 3 above)
Comment (kobertynska) [reply JWPlayer-Developer-Demo/SwiftPlayerViewController.swift]: @AmitaiB I don't like spaces as well. As for including the SDK in the demo, maybe it's better to use CocoaPods or something like that?

**Comment by kobertynska** [comment]
@AmitaiB  I don't like spaces as well. As for including the SDK in the demo, maybe it's better to use CocoaPods or something like that?

**Comment by okulagin** [comment]
@AmitaiB  bump

**Comment by AmitaiB** [comment]
I will update this again branch shortly, and incorporate the feedback from comments there.

**Comment by okulagin** [comment]
@AmitaiB let's avoid the situation when PR was merged when it wasn't approved and changes were required.

**Comment by karimMourra** [comment]
@AmitaiB I agree with @okulagin  we should never merge PRs that have open comments or change requests.

**Comment by AmitaiB** [comment]
@okulagin:  A belated "Oops, that was not my intention."

**Pull request merged**